google-cloud-pubsub/lib/google/cloud/pubsub/project.rb

```diff
@@ -642,13 +647,6 @@ def valid_schema? type, definition, project: nil
         def ensure_service!
           raise "Must have active connection to service" unless service
         end
-
-        ##
-        # Call the publish API with arrays of data data and attrs.
-        def publish_batch_messages topic_name, batch
```

Comment:
Why did we move this method to `batch_publisher` ?

Reply:
This method here was unused hence removed.